Charging.c

```diff
@@ -0,0 +1,73 @@
+#include <stdio.h>
+#include <math.h>
+#include <string.h>
+
+#include "Charging.h"
+
+int chargingSamples[100];
+int NumOfChargingSamples;
+
+void BubbleSort(int chargingSamples[], int NumCS);
+void PrintSortedElements(int chargingSamples[], int NumCS);
+bool AreSampleConsecutive(int chargingSamples[], int NumCS);
+
+int getPeriodicCharingSample()
+{
+  bool consecutive;
+  int i;
+  printf("Enter number of charging samples\n");
+  scanf("%d",&NumOfChargingSamples);
+  
+  printf("Enter the charging samples\n");
+  for(i=0;i<NumOfChargingSamples;i++)
+  {
+    scanf("%d",&chargingSamples[i]);
```

Comment:
Input of values can be abstracted to a different method! this method is difficult to test since every test scenario needs input from console.

Suggest you to take input as a parameter of this method, and then write tests calling this method with various input.